### PR TITLE
Updating CoreOS image to CoreOS stable 1010.5.0 in Node e2e.

### PR DESCRIPTION
**Pull request by vishh**
For #27383

### CODE REVIEW

File: test/e2e_node/jenkins/jenkins-ci.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@ GCE_HOSTS=
 # To copy an image between projects:
 # `gcloud compute --project <to-project> disks create <image name> --image=https://www.googleapis.com/compute/v1/projects/<from-project>/global/images/<image-name>`
 # `gcloud compute --project <to-project> images create <image-name> --source-disk=<image-name>`
-GCE_IMAGES=e2e-node-ubuntu-trusty-docker10-image,e2e-node-ubuntu-trusty-docker9-image,e2e-node-ubuntu-trusty-docker8-image,e2e-node-coreos-stable20160531-image,e2e-node-containervm-v20160321-image
+GCE_IMAGES=e2e-node-ubuntu-trusty-docker10-image,e2e-node-ubuntu-trusty-docker9-image,e2e-node-ubuntu-trusty-docker8-image,e2e-node-coreos-stable20160622-image,e2e-node-containervm-v20160321-image
```
Comment (yifan-gu) [inline test/e2e_node/jenkins/jenkins-ci.properties]: Would be possible to name (or include) the coreos os release number (1010-5-0)?

Comment (vishh) [inline test/e2e_node/jenkins/jenkins-ci.properties]: Can we open an issue for that and fix it in a separate PR?

**Comment by yifan-gu** [comment]
:( It's still failing...

**Comment by yifan-gu** [comment]
I ran again against the coreos stage 1010.5.0, it passed.

File: test/e2e_node/jenkins/jenkins-ci.properties (hunk 1 above)
Comment (yifan-gu) [inline test/e2e_node/jenkins/jenkins-ci.properties]: Sure

**Comment by yujuhong** [comment]
@k8s-bot node e2e test issue: #IGNORE

**Comment by yujuhong** [comment]
@k8s-bot node e2e test this issue: #IGNORE

**Comment by yifan-gu** [comment]
LGTM

**Comment by euank** [comment]
LGTM, thanks!

**Comment by yujuhong** [comment]
@k8s-bot test this issue: #22655

**Comment by vishh** [comment]
@k8s-bot test this github issue #IGNORE

**Comment by euank** [comment]
@k8s-bot test this issue: #27462

**Comment by yifan-gu** [comment]
@k8s-bot test this issue: #27462

**Comment by euank** [comment]
It's concerning that it was the CoreOS image that failed. We should definitely dig into the artifacts (maybe see if it hit the veth bug by looking at dmesg or such). (If no one else does, I'll be able to after lunch)

**Comment by yifan-gu** [comment]
Well, from the latest node e2e log, the tests passed on coreos. It has several failures on ubuntu trusty

**Comment by yifan-gu** [comment]
@k8s-bot test this please

**Comment by euank** [comment]
Based on the kern.log in https://console.cloud.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/27913/node-pull-build-e2e-test/11774/artifacts/tmp-node-e2e-2048fb55-e2e-node-coreos-stable20160622-image/ it was indeed the `unregister_netdevice: waiting for veth2df73ed to become free. Usage count = 1` bug. *sigh*

**Comment by vishh** [comment]
Woot. It finally passed. On the other hand, I just hope that the new CoreOS image does not introduce new flakes.

**Comment by yujuhong** [comment]
> Woot. It finally passed. On the other hand, I just hope that the new CoreOS image does not introduce new flakes.

Has the autoupdate function has been disabled? Just checking the obvious thing since I don't know what else we did for coreos images previously.

**Comment by yifan-gu** [comment]
@yujuhong What autoupdate function do you mean?

**Comment by euank** [comment]
If `setup_host.sh` was run, it disables both update_engine and locksmith, so updates will not be downloaded nor applied. I confirmed with @yifan-gu that he ran that, so it should be happy.

**Comment by yifan-gu** [comment]
> If setup_host.sh was run, it disables both update_engine and locksmith, so updates will not be downloaded nor applied. I confirmed with @yifan-gu that he ran that, so it should be happy.

Yes.

**Comment by alex-mohr** [comment]
Given the changes are to jenkins configs, I don't think retest will actually add more confidence to correctness so flagged with retest-not-required.

**Comment by yifan-gu** [comment]
The coreos node e2e seems flaky. Saw a bunch of PR failed the node e2e tests on coreos.

**Comment by yujuhong** [comment]
Examples:
https://console.cloud.google.com/storage/kubernetes-jenkins/pr-logs/pull/28109/node-pull-build-e2e-test/12216/
https://console.cloud.google.com/storage/kubernetes-jenkins/pr-logs/pull/28025/node-pull-build-e2e-test/12221/

Other than #27498, there are other strange docker error messages, and I filed #28118

**Pull request merged**